src/Notifications.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+
+import { Box, IBox } from './Box';
+import { Flex } from './Flex';
+import { Icon } from './Icon';
+import { Text } from './Text';
+import { iconMap, INotification, NotificationType, Toast } from './Toast';
+
+export interface INotifications extends IBox<HTMLElement> {
+  notifications: INotification[];
+  onMinimize: () => {};
+  onClose: (notification: INotification[]) => {};
```

Comment:
As above, would rather have void than empty block.

```suggestion
  onClose(notification: INotification[]): void;
```

Reply:
BTW, any reason for storing a single notification as an array?